fedmsg/meta/__init__.py

```diff
@@ -135,19 +137,23 @@ def msg2processor(msg, **config):
         return processors[-1]  # DefaultProcessor
 
 
-def legacy_condition(cls):
+def graceful(cls):
```

Comment:
Is there any chance that someone's importing this elsewhere? I don't think it's "officially" documented as part of the API, but I don't see a reason to rename the decorator.

Reply:
Surely there's a chance, but I think it is a low chance.

The `legacy_condition` name is kind of confusing.  "graceful" fits better (as in "graceful degratation").

Reply:
I don't have strong opinions about the name, but if you're going to change it,  it wouldn't hurt to add
```
legacy_condition = graceful
```

and document it as deprecated. There's no reason to break the API, even if it's not "official". It can then be removed in a major release.